EventListener/VersionListener.php

```diff
@@ -53,16 +53,18 @@ public function onKernelRequest(GetResponseEvent $event)
         }
 
         $this->version = $this->versionResolver->resolve($request);
-        if (false === $this->version && null !== $this->defaultVersion) {
+        if (false === $this->version && null !== $this->defaultVersion && !$request->attributes->has('version')) {
             $this->version = $this->defaultVersion;
         }
```

Comment:
Oh, I missed the fact that we made it internal in 1.8. So yes, I would update it accordingly. But we can do that in a separate PR.